clients/admin-ui/src/features/dataset/constants.ts

```diff
@@ -29,3 +29,5 @@ export const FIELD = {
       "Arrays of Data Category resources, identified by fides_key, that apply to this field.",
   },
 };
+
+export const URN_SEPARATOR = "/";
```

Comment:
Nice quick solution changing the separator to something different. 
Although this makes me wonder, could we expect to have "/" as a part of the name too? 😂 

Anyways, we shouldn't have accepted dots as part of the name either, and kirk is working on a fix for that.

Reply:
Yes, it's party why I made this a constant we can update. We can come back and tweak if it's still a problem for other data.